.circleci/config.yml

```diff
@@ -58,27 +58,27 @@ jobs:
       - run: "docker push expressgateway/express-gateway:${CIRCLE_TAG}"
       - run: "docker push expressgateway/express-gateway:latest"
 
+  packer:
+    docker:
+      - image: hashicorp/packer:light
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run: packer build ./.circleci/packer.json
```

Comment:
Do you think is a good idea to put `packer.json` in `.circleci` directory? Maybe a new directory called `release` with all Dockerfiles and `packer.json` file as well?

Reply:
I think the `release` directory is a good idea.  Just make sure it's in `.npmignore` and `.dockerignore`.

Reply:
Will do, thanks!